lisa/sut_orchestrator/azure/features.py

```diff
@@ -783,3 +790,126 @@ def get_azure_disk_type(disk_type: schema.DiskType) -> str:
     schema.DiskType.StandardHDDLRS: "Standard_LRS",
     schema.DiskType.StandardSSDLRS: "StandardSSD_LRS",
 }
+
+
+class Resize(AzureFeatureMixin, features.Resize):
+    def resize(self, resize_action: ResizeAction = None) -> None:
+        platform: AzurePlatform = self._platform
+        compute_client = get_compute_client(platform)
+        vm_operations = compute_client.virtual_machines
```

Comment:
The name is confusing. either remove this line, and call `compute_client.virtual_machines` in later code, or give it a better name.

Reply:
done

Reply:
done

Reply:
The `vm_operations` should be more accurate. not seeing it's changed.

Reply:
I had removed it in one method and forgot to remove in the original. It is now removed in both